front-end/NewLoan/View.js

```diff
@@ -0,0 +1,217 @@
+import React, { Component } from 'react';
+import {
+    AppRegistry,
+    StyleSheet,
+    Text,
+    TextInput,
+    View,
+    Platform,
+} from 'react-native';
+import { Header, Button, Icon } from 'react-native-elements';
```

Comment:
@amelieykw use this library instead, this is what all the projects will use: https://docs.nativebase.io/

Reply:
? I asked you this morning in slack. Can we use this one https://react-native-elements.github.io/react-native-elements/? You said that we can.

Which one exactly should we use?